package.json

```diff
@@ -38,8 +38,12 @@
     "win32",
     "docs/docson/build-schema.json",
     "lib",
-    "scripts"
+    "cli"
   ],
+  "exports": {
+    "./bin_path": "./cli/bin_path.js",
+    "./package.json": "./package.json"
```

Comment:
This is only used once in `.github/workflows/prepare_package_upload.js`.
I'd prefer to keep `.github/workflows/prepare_package_upload.js` as it was and just export `bin_path` here.

Reply:
If no consumers exist, it's better to minimize the API exposure. I can convert this back to an import map.

Reply:
👍